packages/neon_framework/lib/src/bloc/result.dart

```diff
@@ -1,6 +1,12 @@
 import 'package:flutter/widgets.dart';
 import 'package:rxdart/rxdart.dart';
 
+/// An extension that calls [Result.transform] on every [Result] emitted.
+extension ResultStreamTransform<T> on Stream<Result<T>> {
+  /// Transforms every [Result] in the `Stream` using [Result.transform].
+  Stream<Result<R>> transformResult<R>(R Function(T data) callback) => map((e) => e.transform(callback));
+}
+
```

Comment:
Not used in production so either `@visibleForTesting` or move it into the test itself